api/src/main/java/org/eclipse/microprofile/config/ConfigAccessorBuilder.java

```diff
@@ -88,23 +88,23 @@
     ConfigAccessorBuilder<T> cacheFor(Duration duration);
 
     /**
-     * Whether to evaluate variables in configured values.
+     * Whether to expand variables in configured values.
      * A variable starts with '${' and ends with '}', e.g.
      * <pre>
      * mycompany.some.url=${myserver.host}/some/path
      * myserver.host=http://localhost:8081
      * </pre>
-     * If 'evaluateVariables' is enabled, the result for the above key
+     * If 'expandVariables' is enabled, the result for the above key
```

Comment:
-1 on this change. 

I thought about 'evaluateVariables' vs 'expandVariables' for over a month now.
Both are ok-ish, but I don't see how evaluateVariables is wrong!
It explains exactly what is happening. Take a configured value and evaluate all ${} variables in it.
This is also a well known feature from MyFaces CODI, DeltaSpike-Config and Tamaya where exactly this term is being used. So if we want those users to easily migrate over then we should not change it if there is no really good argument.

Reply:
as I said in 
https://github.com/eclipse/microprofile-config/pull/405#discussion_r259766498, I don't mind the name as long as it is consistently used throughout the API and spec